pmoves/tools/unsloth_finetune.py

```
@@ -0,0 +1,111 @@
+    parser.add_argument("--batch-size", type=int, default=2)
+    parser.add_argument("--max-seq-length", type=int, default=8192)
+    parser.add_argument("--rank", type=int, default=16, help="LoRA rank")
+    parser.add_argument("--dry-run", action="store_true", help="Print config without training")
+    args = parser.parse_args()
+
+    try:
+        from unsloth import FastLanguageModel
+        from transformers import TrainingArguments
+        from trl import SFTTrainer
+    except ImportError as e:
+        print(f"ERROR: Unsloth not installed. Run: uv pip install unsloth")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)